wails-frontend/src/pages/PasswordPromptHandler/SignComponentUtils/OperationCost.tsx

```diff
@@ -33,26 +38,34 @@ export interface OperationCostProps {
   isEditing?: boolean;
   setIsEditing: (isEditing: boolean) => void;
   allowFeeEdition: boolean;
+  DeployedByteCodeSize: number; // for executeSC of type deploySC
+  DeployedCoins: number; // for executeSC of type deploySC, in nanoMAS
```

Comment:
do not use number for MAS amounts.
MAS is uint64, which does not fit in js "safe integer"